src/selectors/metadata.js

```diff
@@ -109,6 +109,18 @@ export const mappedDefinitionsListSelector = memoize(moduleName =>
 	),
 );
 
+export const customMappedDefinitionsListSelector = memoize((moduleName, entityName) =>
+	createSelector(mappedDefinitionsListSelector(moduleName, entityName), attributes =>
```

Comment:
it is not **attributes**, it is **definitions**, be more attentive for parameter names